interface/super/edit_layout_props.php

```diff
@@ -106,10 +106,10 @@ function submitProps() {
   <td>
    <select name='form_columns'>
 <?php
+  echo "<option value=''>" . xlt('Default') . " (4)</option>\n";
   for ($cols = 2; $cols <= 10; ++$cols) {
-    echo "<option value=";
-    echo (($cols == 4) ? "'' selected" : "'$cols'");
-    echo ">$cols</option>\n";
+  	if ($cols == 4) continue;
+    echo "<option value='$cols'>$cols</option>\n";
```

Comment:
`$option = "<option value='$cols'$selected>$cols</option>";`

This retains value for option "4" breaking the original (and generic) logic that form fields with blank values be ignored / considered default.  I think 2nd commit makes it clearer to new user by specifying word "Default" like subsequent item.  Otherwise for inexperienced user changing columns setting to 4 system would appear to do nothing.